Sources/SPMBuildCore/BuildParameters/BuildParameters+Testing.swift

```diff
@@ -113,18 +109,14 @@ extension BuildParameters {
             }
         }
 
-        /// Which testing library to use for this build.
-        public var library: Library
```

Comment:
I should probably move the `Library` type out of `BuildParameters` (since it ain't one anymore) but I'm going to save that for a separate PR due to its potential impact to Xcode 16 betas.

Reply:
#7778